.github/workflows/iroha2.yml

```diff
@@ -23,8 +23,26 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - name: Build with Gradle
-        run: ./gradlew build
+      - name: Build with Gradle & Sonarqube analysis
+        run: |
+          ./gradlew build
+          ./gradlew jacocoTestReport
```

Comment:
is `test` executed implicitly?

Reply:
`gradlew tests` is included into `gradlew build` @arndey